modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/constants/UserAssociatedDataWebKeys.java

```diff
@@ -19,6 +19,9 @@
  */
 public class UserAssociatedDataWebKeys {
 
+	public static final String MANAGE_UAD_APPLICATIONS_SUMMARY_DISPLAY =
+		"MANAGE_UAD_APPLICATIONS_SUMMARY_DISPLAY";
```

Comment:
The keys should have consistent naming. Since we use USER_ASSOCIATED_DATA in other keys, we should do the same for this one. Can you change it to `MANAGE_USER_ASSOCIATED_DATA_APPLICATIONS_SUMMARY_DISPLAY`?